### DISCUSSION TITLE
Unable to install Homebrew on my M2 MacBook Air after factory resetting it

### DISCUSSION THREAD

**Discussion by knwn-gzl** [Homebrew/discussions#4011] [Getting started]
When I copy the installation script to the terminal , it displays this error message : 

curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 75220 ms: Operation timed out
I then tried accessing https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh using my browser but it once again gives the same error i.e "Operation Timed Out"
This is my personal macbook air (I myself am the admin) and the network I use is my personal WiFi Network. 

The installation worked when I first logged into this laptop but I am unable to install Homebrew after I reset it to factory settings last week. I already have the Command Line Tools, Vscode and python3 installed. For now I also have Django , Pandas , Numpy and Scikit installed (I used pip for this).

Please help me resolve this issue as I need Homebrew too

**Comment by SMillerDev** [comment]
This is a network issue between you and GitHub. Nothing Homebrew can do

**Reply by knwn-gzl** [reply]
Anything I can do to resolve this tho? I have been trying to install this for a week now and I have tried every available solution I could find on StackOverflow but to no avail

**Reply by p-linnane** [reply]
Unfortunately, this is outside the scope of things we can assist with. It has nothing to do with Homebrew itself, and 100% about your network connection to GitHub. Check your DNS, host based firewall, etc.

**Reply by knwn-gzl** [reply]
Thanks for the assistance, I will go check my DNS settings

**Reply by Mohitlekhwar** [reply]
You can enter on the network section or add new DNS like google: 8.8.8.8 then restart your network setting.
